airflow/api_fastapi/serializers/dag_run.py

```diff
@@ -18,13 +18,28 @@
 from __future__ import annotations
 
 from datetime import datetime
+from enum import Enum
 
 from pydantic import BaseModel, Field
 
 from airflow.utils.state import DagRunState
 from airflow.utils.types import DagRunTriggeredByType, DagRunType
 
 
+class DAGRunModifyFormStates(str, Enum):
```

Comment:
I think we can drop the `Form` in the name. This is not called from an http form anymore. (I guess this is where the name is comming from)
```suggestion
class DAGRunModifyStates(str, Enum):
```